modin/core/execution/dask/implementations/pandas_on_dask/partitioning/virtual_partition.py

```diff
@@ -483,6 +501,10 @@ class PandasOnDaskDataframeColumnPartition(PandasOnDaskDataframeVirtualPartition
         Whether this partition spans an entire axis of the dataframe.
     call_queue : list, optional
         A list of tuples (callable, args, kwargs) that contains deferred calls.
+    length : distributed.Future or int, optional
```

Comment:
Can we `inherit_docstrings` from parent class?

Reply:
let's try